frontend/src/pages/admin/FlowBlueprints/index.vue

```diff
@@ -150,7 +181,33 @@ export default {
                 }
             })
             this.teamTypes.sort((A, B) => { return A.order - B.order })
+        },
+        exportFlowBlueprints () {
+            FlowBlueprintsApi.exportFlowBlueprints()
+                .then((data) => {
+                    const filename = `blueprints_export_${dateToSlug(new Date())}.json`
+                    downloadData(data.blueprints, filename)
+                })
+                .catch(err => console.error(err.message))
+        },
+        onImportFlowBlueprints (blueprints) {
+            FlowBlueprintsApi.importFlowBlueprints(blueprints)
+                .then(response => this.setBlueprints(response.blueprints))
+                .then(() => {
+                    Alerts.emit('Blueprints successfully imported!', 'confirmation', 7500)
+                })
+                .catch(err => {
+                    Alerts.emit('Something went wrong!', 'warning', 7500)
+                    console.error(err.message)
+                })
         }
     }
 }
 </script>
+
+<style lang="scss">
```

Comment:
I assumed that since the .tools class is added via template in the SectionTopMenu component, the style tag would need to be un-scoped but It seems to be working with it being scoped.